src/libraries/System.DirectoryServices.AccountManagement/tests/UserPrincipalTest.cs

```diff
@@ -28,33 +28,5 @@ public void UserPrincipalConstructorTest()
             UserPrincipal user = new UserPrincipal(DomainContext);
             user.Dispose();
         }
-
-        public void ComputedUACCheck()
```

Comment:
dead code

Reply:
Let me correct that: it's a test of AuthenticablePrincipal.IsAccountLocked out, but it never got a `[Fact]` on it. If we bring a test back, we can reuse code then.

Reply:
I'm deleting this one and not the other one as this seems like something we'd always keep as a manual test.